websites/S/SuperiorServers.co/presence.ts

```diff
@@ -0,0 +1,177 @@
+const presence = new Presence({
+    clientId: "792094839414980639"
+});
+
+presence.on("UpdateData", async () => {
+  const currentTime = Math.floor(Date.now() / 1000), presenceData: PresenceData = {
+    largeImageKey: "main",
+    startTimestamp: currentTime
+  };
+  if (document.location.hostname == "superiorservers.co") {
+    presenceData.largeImageKey = "main";
+    presenceData.details = "Portal";
+    if (document.location.pathname == "/" || !document.location.pathname) {
+        presenceData.startTimestamp = currentTime;
+    }
+    if (document.location.pathname.includes("/staff")){
+      let DRPStaff = document.querySelector("#RP_info").textContent;
+      DRPStaff = DRPStaff.substr((DRPStaff.indexOf(" of") + 3), DRPStaff.indexOf(" entries")).replace(" entries", "");
+      let SWRPStaff = document.querySelector("#SWRP_info").textContent;
+      SWRPStaff = SWRPStaff.substr((SWRPStaff.indexOf(" of") + 3), SWRPStaff.indexOf(" entries")).replace(" entries", "");
+      let MilRPStaff = document.querySelector("#MilRP_info").textContent;
+      MilRPStaff = MilRPStaff.substr((MilRPStaff.indexOf(" of") + 3), MilRPStaff.indexOf(" entries")).replace(" entries", "");
+      let AllServersStaff = document.querySelector("#NO_ID_info").textContent;
+      AllServersStaff = AllServersStaff.substr((AllServersStaff.indexOf(" of") + 3), AllServersStaff.indexOf(" entries")).replace(" entries", "");
+      const totalStaff = (parseInt(DRPStaff) + parseInt(SWRPStaff) + parseInt(MilRPStaff) + parseInt(AllServersStaff)).toString();
+      presenceData.details = "Viewing the Staff list";
+      presenceData.state = totalStaff + " total members";
+      presenceData.startTimestamp = currentTime;
+    }
+    if (document.location.pathname.includes("/bans")){
+      let numBans = document.querySelector("div.dataTables_info").textContent;
+      const start = numBans.indexOf("of ") + 3, end = numBans.indexOf(" entries");
+      numBans = numBans.substr(start, end).replace(" entries", "");
+      presenceData.details = "Viewing " + numBans + " bans";
```

Comment:
please change it to const and do substr after textContent because you can do it obviously